Sensor.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Sensor.h"
+
+namespace LFRobot
+{
+	Sensor::Sensor(int inPin)
+		: pin(inPin) // pin is given the value of the argument
+	{
+		pinMode(pin, INPUT);
+
+	}
+
+	void Sensor::setMode(SensorMode mode)
+	{
+		if (mode == OUT)
+		{
+			pinMode(pin, OUTPUT);
+			digitalWrite(pin, HIGH);
```

Comment:
If `Sensor` is only going to be used with the Reflectance Sensor Array then this is fine. However if `Sensor` is to be reused by other sensors types in the project, defaulting the output to HIGH is risky.